### DISCUSSION TITLE
Change chat port is possible?

### DISCUSSION THREAD

**Discussion by TagorePde** [Ylianst/MeshCentral#6217] [Q&A]
When i tried to open a chat to a computer, the url dont have any port, and is not working, if i edited manually the url, and add the 8086 port, is working, can i change that? 
i access MC with something like this: https://mc.domain.tld:8086

my config:

```
{
   "$schema":"http://info.meshcentral.com/downloads/meshcentral-config-schema.json",
   "settings":{
      "cert":"mc.EDITED.tld",
      "_WANonly":true,
      "_LANonly":true,
      "sessionKey":"EDITED",
      "port":443,
      "AgentPort":8086,
      "_aliasPort":8086,
      "redirPort":80,
      "_redirAliasPort":80,
      "AgentPong":300,
      "TLSOffload":false,
      "SelfUpdate":false,
      "AllowFraming":"false",
      "WebRTC":"false",
      "AutoBackup":{
         "backupPath":"/opt/meshcentral/meshcentral-backups",
         "backupInvervalHours":24,
         "keepLastDaysBackup":"10",
         "zippassword":"EDITED"
      }
   },
   "domains":{
      "":{
         "_title":"MyServer",
         "_title2":"Servername",
         "_minify":true,
         "Title":"B&V",
         "Title2":"Remote",
         "TitlePicture":"BVRemote.png",
         "siteStyle":"2",
         "nightMode":"1",
         "NewAccounts":"true",
         "_userNameIsEmail":true,
         "_certUrl":"my.reverse.proxy",
         "allowedOrigin":true,
         "consentMessages":{
            "Title":"BV"
         },
         "notificationMessages":{
            "Title":"BV"
         }
      }
   },
   "letsencrypt":{
      "__comment__":"Requires NodeJS 8.x or better, Go to https://letsdebug.net/ first before>",
      "email":"sistemas@EDITED.tld",
      "names":"mc.EDITED.tld",
      "production":true
   }
}
```

**Comment by si458** [comment]
You have port set to 443 so it's trying to use that port
I'm not sure why entering port 8086 is working as that is the agent port which doesn't work for the chat?
You also have tlsoffload set as false which doesn't seem right either, m

**Comment by si458** [comment]
You should remove the agentport value, and set port to 8086, and also remove the tlsoffload as this is only used if u have a reverse proxy

**Reply by TagorePde** [reply]
Hi, thanks for the answer, tested, and nothing is working ... rolled back.

When i press to chat with some of the computers, the admin get the url with 8086 port, and is working, is the page opened in the remote computer that not have the port in the url and is not working, if i change the url adding the port 8086 is working, but the remote user dont know that ... 
Regards.

also thank for editing my message, i was unable to do proper format of the json part.

**Reply by si458** [reply]
So do u access ur Web ui for meshcentral using `https://mc.domain.tld`?

And then ur agents connect to `https://mc.domain tls:8086`?

And u also dont use a reverse proxy like nginx/traefik/cloudflare?

If that's the case then u should remove the tlsoffload option as that's only needed for reverse proxies

But this shouldn't effect the chat functionality?

It might be missing the agent port off when asking the remote device to open the chat

Can u clarify what devices do/don't work?

What OS do open correctly?

What OS don't open correctly?
